data-pipeline.yml

```diff
@@ -17,6 +17,7 @@ tasks:
     routes:
       - notify.email.release-mgmt-analysis@mozilla.com.on-failed"
       - notify.irc-channel.#bugbug.on-failed
+      - index.project.relman.bugbug.bugbug_data.latest
```

Comment:
Nit: given that we have project.relman.bugbug.XXX, we can avoid rewriting bugbug again. So this could be something like: index.project.relman.bugbug.data_bugzilla.latest

Reply:
Similarly with the other index names